<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportCertDlg</class>
 <widget class="QDialog" name="ImportCertDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import certificate</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Label</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QComboBox" name="mSensitiveCombo"/>
   </item>
   <item row="10" column="0">
    <widget class="QCheckBox" name="mTokenCheck">
     <property name="text">
      <string notr="true">Token</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="mSessionText"/>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="4">
    <widget class="QLineEdit" name="mCertPathText"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Attributes</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="5">
    <widget class="QLineEdit" name="mIDText"/>
   </item>
   <item row="12" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Subject</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Slot status</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Cert Path</string>
     </property>
    </widget>
   </item>
   <item row="9" column="4">
    <widget class="QCheckBox" name="mModifiableCheck">
     <property name="text">
      <string notr="true">Modifiable</string>
     </property>
    </widget>
   </item>
   <item row="13" column="3" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Slot ID</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Session</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QComboBox" name="mSlotsCombo"/>
   </item>
   <item row="3" column="5">
    <widget class="QLineEdit" name="mLoginText"/>
   </item>
   <item row="9" column="2">
    <widget class="QCheckBox" name="mSensitiveCheck">
     <property name="text">
      <string notr="true">Sensitive</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="mSlotIDText"/>
   </item>
   <item row="7" column="5">
    <widget class="QPushButton" name="mFindBtn">
     <property name="text">
      <string>Find</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Slot list</string>
     </property>
    </widget>
   </item>
   <item row="9" column="5">
    <widget class="QComboBox" name="mModifiableCombo"/>
   </item>
   <item row="10" column="1">
    <widget class="QComboBox" name="mTokenCombo"/>
   </item>
   <item row="9" column="1">
    <widget class="QComboBox" name="mPrivateCombo"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="5">
    <widget class="QLineEdit" name="mLabelText"/>
   </item>
   <item row="9" column="0">
    <widget class="QCheckBox" name="mPrivateCheck">
     <property name="text">
      <string notr="true">Private</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="5">
    <widget class="QLineEdit" name="mSubjectText"/>
   </item>
   <item row="11" column="0">
    <widget class="QCheckBox" name="mStartDateCheck">
     <property name="text">
      <string>StartDate</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QCheckBox" name="mEndDateCheck">
     <property name="text">
      <string>EndDate</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QDateEdit" name="mStartDateEdit"/>
   </item>
   <item row="11" column="3">
    <widget class="QDateEdit" name="mEndDateEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImportCertDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImportCertDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
